src/commands/iota/create-pex-query.ts

```diff
@@ -0,0 +1,90 @@
+import { readFile } from 'fs/promises'
+import { CreatePexQueryInput, PexQueryDto } from '@affinidi-tdk/iota-client'
+import { ux, Flags } from '@oclif/core'
+import { CLIError } from '@oclif/core/errors'
+import z from 'zod'
+import { BaseCommand } from '../../common/base-command.js'
+import { giveFlagInputErrorMessage } from '../../common/error-messages.js'
+import { promptRequiredParameters } from '../../common/prompts.js'
+import { INPUT_LIMIT, PRESENTATION_DEFINITION_LIMIT, validateInputLength } from '../../common/validators.js'
+import { iotaService } from '../../services/affinidi/iota/service.js'
+
+export class CreatePexQuery extends BaseCommand<typeof CreatePexQuery> {
+  static summary = 'Creates PEX query for your Iota configuration'
+  static examples = [
+    '<%= config.bin %> <%= command.id %> -i <value> -n <value> -d <value> -f pexQuery.json',
+    '<%= config.bin %> <%= command.id %> --configuration-id <value> --name <value> --description <value> --file pexQuery.json',
+  ]
+
+  static flags = {
+    'configuration-id': Flags.string({
+      char: 'i',
```

Comment:
A bit subjective, but I don't like using `i` for configuration in query commands. It can be confused with the query id. Maybe use `c` for configuration id and `i` for query id?